hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/scope/HandleHederaOperations.java

```diff
@@ -195,7 +200,15 @@ public void updateStorageMetadata(
     @Override
     public void createContract(
```

Comment:
A comment that describes this use cases (which I think is it derives the new contract from the parent) vs one below would be nice

Reply:
The javadoc on the `HederaOperations` interface describes this difference.